examples/shared-vpc-with-internal-load-balancer/README.md

```diff
@@ -44,8 +63,10 @@ any instances attached to the subnetwork.
 ### Service
 
 The compute resources which comprise the majority of the TFE Clustered
-infrastructure are created in the service project as with a single
-project deployment.
+infrastructure are created in the service project, similar to a
+single-project deployment. The notable exception in the use of the
```

Comment:
```suggestion
single-project deployment. The notable exception is in the use of the
```

small sentence fix, and also

do we want to have a small explanation about what this exception actually entails?

Reply:
I've tried to address this with https://github.com/hashicorp/terraform-google-terraform-enterprise/pull/74/commits/3804d934cf3e7fe63f56bd17c22407a39e329a83. What do you think?

Reply:
I was more meaning that the way the paragraph reads, is that all the resources are in a single project, and that the exception to all being in a single project was in the use of both or either of the load balancer modules, ie. that the load balancer resources would not be in the same, single project.

if this is the case, just a little detail on like what that means in the context of the project locality of the deployment and like if we're creating a new namespace for the load balancer deployment or if it expects an existing one?

Reply:
Ooh sorry. Yeah, it's not supposed to read like that. The exception is the use of the internal load balancer rather than the external load balancer.

Reply:
@erindatkinson how about https://github.com/hashicorp/terraform-google-terraform-enterprise/pull/74/commits/d456910c5dc63cf9b70457efa8ed16920e3b7a1d?